### PR TITLE
fixed the outputs of the --versions options on 3 scripts

### LINKED ISSUE #1860: Wrong Plugin Returns
Hi,

i suddenly discovered that some monitoring-plugins have wrong return codes on the version output. Wrong as in "doesnt correspond with the guidelines in https://www.monitoring-plugins.org/doc/guidelines.html#AEN78"

This happens at least in check_sensors, check_oracle and check_log.

### PR DESCRIPTION
**Pull request by MisterMountain**
This fixes #1860

### CODE REVIEW

File: plugins-scripts/check_oracle.sh [hunk 1]
```diff
@@ -95,7 +95,7 @@ case "$cmd" in
     ;;
 --version)
 		print_revision "$PROGNAME" "$REVISION"
-    exit "$STATE_OK"
+    exit "$STATE_UNKNOWN"
```
Comment (RincewindsHat) [inline plugins-scripts/check_oracle.sh:98]: While you are at it, could you fix the indentation here? :-)

**Review by RincewindsHat — CHANGES_REQUESTED** [review]
Hi @MisterMountain,
Thanks very much. One small change would be welcome though.

**Review by dermoth — APPROVED** [review]
Looks good to me, I'll defer merging to active maintainers.

GH doesn't let me see the ws changes (i.e. tabs vs spaces) and typically I believe we used to standardize to tabs,, this appears to be using spaces. IMHO consistency within a single script is much more important so I have no concerns there.

**Review by dermoth — APPROVED** [review]

**Comment by RincewindsHat** [comment]
@dermoth you really like that approve-button, don't you? :-)

File: plugins-scripts/check_oracle.sh [hunk 2]
```diff
@@ -86,20 +86,20 @@ esac
...
 case "$cmd" in
 --help)
-		print_help
+    print_help
     exit "$STATE_OK"
     ;;
 -h)
-		print_help
+    print_help
     exit "$STATE_OK"
```
[the remark this answers is not in the document]
Comment (dermoth) [reply plugins-scripts/check_oracle.sh:94]: Actually I missed that (thought it was corrected in later commits)

Please fix

I haven't used GH for too long I guess.

**Review by dermoth — CHANGES_REQUESTED** [review]

**Comment by RincewindsHat** [comment]
Actually since @MisterMountain mentioned, that the exit code for `help` and `version` should be `UNKNOWN` I was hoping, that the corresponding exit codes would also be fixed. This occured to me when I reviewed the changes. For example the exit code for `check_log` and `help` is `OK`

**Comment by MisterMountain** [comment]
@RincewindsHat fixing the exit codes for help is still on my todolist :)

**Comment by MisterMountain** [comment]
It took a while, but i did these changes - is there still something i forgot?

**Review by RincewindsHat — APPROVED** [review]
Looks good to me

**Pull request merged**